test/clustertest/testplugin/TestPlugin.cpp

```diff
@@ -217,6 +245,19 @@ bool TestPluginCommand::peek(SQLite& db) {
             // case
             return false;
         }
+    } else if (request.methodLine == "testescalate") {
+        string serverState = plugin().server.getState() == SQLiteNode::LEADING ? "LEADER" : "FOLLOWER";
+        string statString = "Peeking testescalate (" + serverState + ")\n";
+        if (serverState == "FOLLOWER") {
+            // When peeking on a follower (which should be the first thing we do), let's create the temp file.
+            string tempFile = BedrockTester::getTempFileName("escalate_test");
```

Comment:
couldn't we generate this file in the test and add the file name as a request param? that would avoid the `const_cast`